### DISCUSSION TITLE
Getting poetry.lock outside of Git

### DISCUSSION THREAD

**Discussion by YogevBokobza** [TomerFi/aioswitcher#738] [Ideas]
Hi @TomerFi,

Is it possible to remove poetry.lock file and move it to .gitignore?
This file is always getting messed up for me in PRs..

Yogev.

**Comment by TomerFi** [comment]
[Looks like it](https://python-poetry.org/docs/basic-usage/#committing-your-poetrylock-file-to-version-control). But I haven't tried it. I don't mind either way—it's your call.

**Comment by thecode** [comment]
I would add it to ignore, there is no need for it to be updated from user branch

**Reply by YogevBokobza** [reply]
@thecode That's my thinking as well

**Comment by YogevBokobza** [comment]
Already in master. Can be closed.
